### PR TITLE
Created database scripts for movie db

### PR DESCRIPTION
**Pull request by riteshdalvi270**

### CODE REVIEW

File: liveIT-db/schema/movies/Movies.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+type varchar(50) not null,
+releasedDate date not null,
+watchDate date default now(),
+venueId smallint not null,
+amountPayed decimal(13,2) not null,
+screenMode varchar(10) not null default '2D',
+ourRating smallint,
+genreId smallint,
+imdbLink varchar(500),
+rottenTomatoesLink varchar(500),
```
Comment (ridonkz) [inline liveIT-db/schema/movies/Movies.sql]: the correct term is URL!!

File: liveIT-db/schema/movies/Movies.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+name varchar(50) not null,
+type varchar(50) not null,
+releasedDate date not null,
+watchDate date default now(),
+venueId smallint not null,
+amountPayed decimal(13,2) not null,
+screenMode varchar(10) not null default '2D',
+ourRating smallint,
+genreId smallint,
+imdbLink varchar(500),
```
Comment (ridonkz) [inline liveIT-db/schema/movies/Movies.sql]: correct term is URL!1

File: liveIT-db/schema/movies/Movies.sql (hunk 1 above)
Comment (riteshdalvi270) [reply liveIT-db/schema/movies/Movies.sql]: Done

File: liveIT-db/schema/movies/Movies.sql (hunk 2 above)
Comment (riteshdalvi270) [reply liveIT-db/schema/movies/Movies.sql]: Done

**Review by ridonkz — APPROVED** [review]

**Pull request merged**